### PR TITLE
New admin endpoint: returns last completed deploys

### PR DESCRIPTION
**Pull request by TBonnin**
## What does this change?
I am gonna experiment with Prout. This patch is duplicating an endpoint in admin that returns all completed deploys made by riffraff (with their commit sha), which Prout would check to determine if a PR has been deployed or not.

If the experiment is successful, I would then cleanup all the `deploys-radiator` and `deploys-notify` endpoints which wouldn't be necessary anymore

### CODE REVIEW

File: admin/app/model/deploys/RiffRaff.scala [hunk 1]
```diff
@@ -6,26 +6,31 @@ import model.deploys.ApiResults.{ApiError, ApiErrors, ApiResponse}
 import play.api.libs.json.{JsError, JsSuccess, Json}
 import scala.concurrent.Future
 
+case class RiffRaffDeployTags(vcsRevision: String)
+object RiffRaffDeployTags { implicit val format = Json.format[RiffRaffDeployTags] }
```
Comment (jfsoul) [inline admin/app/model/deploys/RiffRaff.scala]: Not that it matters much, but could this just be a `Json.reads`?

File: admin/app/controllers/DeploysRadiatorController.scala [hunk 2]
```diff
@@ -16,8 +16,12 @@ trait DeploysRadiatorController extends Controller with Logging with Requests {
   val riffRaff: RiffRaffService
   val teamcity: TeamcityService
 
-  def getDeploys(pageSize: Option[String], projectName: Option[String], stage: Option[String]) = Action.async {
-    riffRaff.getRiffRaffDeploys(pageSize, projectName, stage).map(ApiResults(_))
+  def getCompletedDeploys(pageSize: Option[String], projectName: Option[String], stage: Option[String]) = Action.async {
```
Comment (jfsoul) [inline admin/app/controllers/DeploysRadiatorController.scala]: Is there much point in keeping `projectName` as an argument? Could hardcode to dotcom:all

**Review by jfsoul — APPROVED** [review]
Looks good enough to go 👍

File: admin/app/model/deploys/RiffRaff.scala (hunk 1 above)
Comment (TBonnin) [inline admin/app/model/deploys/RiffRaff.scala]: It is used by the `Json.format[RiffRaffDeploy]` below so yes we need a format

File: admin/app/controllers/DeploysRadiatorController.scala (hunk 2 above)
Comment (TBonnin) [inline admin/app/controllers/DeploysRadiatorController.scala]: In theory yes, I didn't want to change to much while I am still experimenting. I am happy to reconsider the `projectName` argument if/when I cleanup

**Pull request merged**